tests/test-data-interface.py

```diff
@@ -1,43 +0,0 @@
-import torch
-from super_gradients.training.datasets.dataset_interfaces import DatasetInterface
-from super_gradients.training.sg_trainer import Trainer
-from torchvision.models import resnet18
-import numpy as np
-
-
-class TestDatasetInterface(DatasetInterface):
```

Comment:
why was that deleted? I see this is imported in the test suit

Reply:
No, it wasn't used anywhere (and had no meaning). You are probably mistaking this with some other tests.